### DISCUSSION TITLE
Kohya's HighRes-Fix for better images

### DISCUSSION THREAD

**Discussion by kubilaykilinc** [lllyasviel/Fooocus#1010] [Ideas]
I saw a video today about Kohya's HighRes-Fix for better images. (There is for Automatic1111 and ComfyUI)

https://www.youtube.com/watch?v=FO7fxrDGXE0

Maybe it can be innovative for Fooocus. 

This is Kohya's HighRes-Fix Python code for ComfyIUI custom node

https://gist.githubusercontent.com/laksjdjf/487a28ceda7f0853094933d2e138e3c6/raw/250227b9d83b8bdba8bbc4fd582bdddb955d3a40/kohya_hiresfix.py

**Comment by Ambratolm** [comment]
Can this make faces and specially Eyes look better in long shot scenes with original resolutions ?

**Comment by W4hr** [comment]
The addition of this feature would be really helpful especially for in- and outpainting and perhaps this can be used with the new SDXL turbo model to produce 1024px images instead of the model's recommended size of 512px by 512px. This would be great as SDXL turbo is not only faster than LCM or the SSD model, at least for me, but also faster than SD 1.5 while giving better results than the 1.5 base model.

SDXL turbo:
[Reddit announcement](https://www.reddit.com/r/StableDiffusion/comments/186496i/introducing_sdxl_turbo_a_realtime_texttoimage/)
[Weights](https://huggingface.co/stabilityai/sdxl-turbo)

**Comment by xhoxye** [comment]
This is very good
